### PR TITLE
KIALI-2913 Add Jaeger documentation to Kiali.io

### PR DESCRIPTION
**Pull request by aljesusg**
JIRA: [KIALI-2913](https://issues.jboss.org/browse/KIALI-2913)

### CODE REVIEW

File: content/documentation/tracing/index.adoc [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+= Tracing your application in Kiali
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Distributed Tracing
+:keywords: Kiali Documentation Distributed Tracing
+:icons: font
+:imagesdir: /images/documentation/tracing/
+
+Kiali show tracing of your applications. We currently support jaeger from the version 1.9 (link:https://www.jaegertracing.io/docs/1.12/frontend-ui/#embedded-mode[Read docs, window="_blank"]).
```
Comment (abonas) [inline content/documentation/tracing/index.adoc]: jaeger - capital J :)

Comment (abonas) [inline content/documentation/tracing/index.adoc]: shows

File: content/documentation/tracing/index.adoc [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+icon:bullhorn[size=2x]{nbsp} In order to have the Jaeger embedded feature, you must have the same protocol in Kiali and Tracing service. These are necessary for Kiali to embed Jaeger.
+
+== Kiali an Tracing Service have the same protocol
```
Comment (abonas) [inline content/documentation/tracing/index.adoc]: and

File: content/documentation/tracing/index.adoc [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+When Kiali and Tracing service have the same protocol we'll have the embedded feature in the website.
```
Comment (abonas) [inline content/documentation/tracing/index.adoc]: protocol, we (add ",")

File: content/documentation/tracing/index.adoc [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+image::service_view.png[Service View]
+
+We can interact with this embedded feature like the Jaeger UI
+
+image::trace_view.png[Trace View]
+
+=== Tracing Toolbar
+
+Kiali have a toolbar to control this feature to facilitate the user experience with the tracing service. When we go the tracing view we'll see a small form as the image below where we can select the service and the lookback(Traces from last minutes or hours).
```
Comment (abonas) [inline content/documentation/tracing/index.adoc]: we go to the tracing view

File: content/documentation/tracing/index.adoc [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+
+* Tags: Empty by default, find the traces with these values. Example: `http.status_code=200`,error=true`
+* Limit results: 20 by default, number of traces results.
+* Span Configurations (More info about spans link:https://www.jaegertracing.io/docs/1.9/architecture/#span[here, window="_blank"])
+** Min Duration: Min duration for a span. Eg: 1.2s, 100ms, 500us
+** Max Duration: Max duration for a span. Eg: 1.2s, 100ms, 500us
+
+
+== Kiali an Tracing Service have not the same protocol
```
Comment (abonas) [inline content/documentation/tracing/index.adoc]: an -> and

File: content/documentation/tracing/index.adoc [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+In this case you'll have a link to open the service in a new tab in your browser in the menu of kiali and in the view of a specific service.
```
Comment (abonas) [inline content/documentation/tracing/index.adoc]: kiali - > capital K

File: content/documentation/tracing/index.adoc [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+Check in the about modal if you have the JaegerURL variable with the URL
+
+image::about_menu.png[About menu]
+
+image::about.png[About modal]
+
+
+
+=== I have not Tracing URL in the about modal.
```
Comment (abonas) [inline content/documentation/tracing/index.adoc]: not -> no, or "I don't have a ..."

File: content/documentation/tracing/index.adoc [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+So Kiali didn't found the tracing service. Kiali tries to search for the tracing service in `tracing` or `jaeger-query`. If you have another name for this service you must set the values in the link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[yaml configuration, window="_blank"] for link:https://operatorhub.io/operator/alpha/kiali-operator[Kiali Operator, window="_blank"]
```
Comment (abonas) [inline content/documentation/tracing/index.adoc]: found -> find

**Comment by JStickler** [comment]
Navigating from the Documentation drop down menu works, but for some reason the link off the Documentation landing page (https://www.kiali.io/documentation/) points to https://www.kiali.io/documentation/**distributed-**tracing instead of to https://www.kiali.io/documentation/tracing/.

**Comment by aljesusg** [comment]
What  link @JStickler ?

**Comment by josejulio** [comment]
I think she is talking about this link:
Screenshot_2019-05-28_14-54-22

Screenshot_2019-05-28_14-55-59

**Comment by JStickler** [comment]
That's the one!

**Comment by aljesusg** [comment]
Fix https://github.com/kiali/kiali.io/pull/129

**Pull request merged**